packages/tc-api-rest-handlers/__tests__/patch-create.spec.js

```diff
@@ -1,766 +1,57 @@
-const { setupMocks, neo4jTest } = require('../../../test-helpers');
-const { dbUnavailable } = require('../../../test-helpers/error-stubs');
+const { setupMocks } = require('../../../test-helpers');
 
-const { patchHandler } = require('../patch');
+jest.mock('../post', () => ({
+	postHandler: jest.fn(),
+}));
+const { patchHandler: patchHandlerFactory } = require('../patch');
 
-describe('rest PATCH create', () => {
```

Comment:
Highlight #8 The tests in general don't stub anything out. However, when PATCHing a record that doesn't already exists it literally routes through ti the PST handler. To avoid the tedious duplication of tests (with the inevitable inconsistencies between what is tested in both cases) the test suite for patching non-existent records stubs out the post handler and just verifies it is called. This feels like a justifiable bit of stubbing.